asset/js/main.js

```diff
@@ -0,0 +1,68 @@
+const regex_check_special_characters =
```

Comment:
```suggestion
const regexCheckSpecialCharacters =
```
Đặt tên biến, tên hàm theo camelCase nhé em